### ISSUE TITLE
Coline in indoor canvas of shmetro

### ISSUE THREAD

**Issue by thekingofcity** [wongchito/RailMapGenerator#279]
> V5中上海版共线门板图未更新（可能应该是还在开发吧）

_Originally posted by @OrdinaryRUNYING in https://github.com/wongchito/RailMapGenerator/issues/276_

**Comment by XHG78999** [comment]
Right, I think we need indoor like 04A01, is it? we're doing same works in railmap!

**Comment by thekingofcity** [comment]
> 这个应该可以被确认为不被支持的功能了，简单来说是由于之前的indoor实现中使用了mtr的线路画法实现，将其改变为上海风格的门板图几乎不可能。所以在门板图被重写前应该是无法看到共线了。虽然我也很讨厌不完美，但受限于精力和时间大抵仅能如此了。🙃
> This will be an unsupported feature for a long time. The reason for short is the reuse of the MTR implementation. Although shortens the development cycle, it is really difficult to comply with the coline layout. So no coline in indoor. 🙃

_Originally posted by @thekingofcity in https://github.com/wongchito/RailMapGenerator/issues/276#issuecomment-1066084377_

**Comment by thekingofcity** [comment]
硬要说的话可以在有coline的时候调用railmap的实现，因为coline的indoor不会展示支线，但当有非共线支线时，这就不work了。因为我要使用mtr的path+hide branch when there is a coline.
